pkg/localize/locales/en/cmd/cluster.en.toml

```diff
@@ -316,7 +316,7 @@ You need to separately grant service account access to Kafka by issuing followin
 
 [cluster.kubernetes.printRegistryAccessCommands]
 one = '''
-You might need to assign non-readonly roles for the service account based on the use case (READ_ONLY by default):
+You need to assign role for the service account:
```

Comment:
I think it is missing something...
Maybe:
- You need to assign the **following role**...
- You should assign the **following role**...
- You should assign the **DEVELOPER role**...

cc @jbyrne-redhat

Reply:
Would `one of the roles for the service account in order to use it with service registry`

Reply:
```suggestion
You need to assign one of the roles for the service account in order to use it with service registry. For example:
```